MyBlog.Services/Helpers/EncryptDecrypt/EncryptDecryptManager.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+
+namespace MyBlog.Service.Helpers.EncryptDecrypt;
+
+public class EncryptDecryptManager: IEncryptDecryptManager
+{
+    private readonly string? _key;
+    
+    public EncryptDecryptManager(IConfiguration config)
+    {
+        _key = config["PasswordKey:Key"];
+    }
+    
+    public string Encrypt(string text)
+    {
+        byte[] aesIv = new byte[16];
+        byte[] array;
```

Comment:
what is aesIv ? i`m other developer, read your code, and actually do not understand